src/App.js

```diff
@@ -0,0 +1,27 @@
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+} from 'react-router-dom';
+import BookList from './Componenets/BookList';
+import BookForm from './Componenets/BookForm';
+import Nav from './Componenets/Nav';
+import Catogory from './Componenets/Catogories';
+
+const App = () => (
+  <>
+    <Router>
+      <Nav />
```

Comment:
Has 2 `<Link>`s in the navbar which leads to the booklist and categories locations 🚀 Well done!